ramls/check-in-by-barcode-response.json

```diff
@@ -78,6 +78,11 @@
             "name": {
               "description": "Name of the item status",
               "type": "string"
+            },
+            "date": {
+              "type": "string",
+              "description": "Date time when status was last changed",
+              "format": "date-time"
```

Comment:
Was this already being provided, and missing in the interface definition, or is it a new property?

Reply:
It is an existing property. It is being set in [ItemSummaryRepresentation.java#L44](https://github.com/folio-org/mod-circulation/blob/master/src/main/java/org/folio/circulation/domain/representations/ItemSummaryRepresentation.java#L44) which called in `org.folio.circulation.domain.representations.CheckInByBarcodeResponse#mapToResponse`